tap_github/streams.py

```diff
@@ -0,0 +1,102 @@
+from enum import Enum
+from typing import Type, Set, List
+
+from singer_sdk.streams.core import Stream
+
+from tap_github.organization_streams import OrganizationStream
+from tap_github.repository_streams import (
+    AnonymousContributorsStream,
+    CommitsStream,
+    CommunityProfileStream,
+    ContributorsStream,
+    EventsStream,
+    IssueCommentsStream,
+    IssueEventsStream,
+    IssuesStream,
+    LanguagesStream,
+    PullRequestsStream,
+    ReadmeHtmlStream,
+    ReadmeStream,
+    RepositoryStream,
+    StargazersStream,
+    StatsContributorsStream,
+    AssigneesStream,
+    CollaboratorsStream,
+    ReviewsStream,
+    ReviewCommentsStream,
+    ProjectsStream,
+    ProjectColumnsStream,
+    ProjectCardsStream,
+    PullRequestCommits,
+    MilestonesStream,
+    CommitCommentsStream,
+    ReleasesStream,
+)
+from tap_github.user_streams import (
+    StarredStream,
+    UserContributedToStream,
+    UserStream,
+)
+from tap_github.organization_streams import (
+    OrganizationStream,
+    TeamsStream,
+    TeamMembersStream,
+    TeamRolesStream,
+)
+
+
+class Streams(Enum):
+    valid_queries: Set[str]
+    streams: List[Type[Stream]]
+
+    def __init__(self, valid_queries: Set[str], streams: List[Type[Stream]]):
+        self.valid_queries = valid_queries
+        self.streams = streams
+
+    REPOSITORY = (
```

Comment:
not too sure what you mean by this. Do you mean repository_streams.py? If you mean splitting up this enum into several places we might lose functionality such as the `all_valid_queries` which reads all possible streams at once

Reply:
I meant that potentially you could create a list called `repository_streams_list` in repository_strems.py and simply import it here. So this file would get simpler because the import would become:

import `repository_streams_list` from repository_streams

And the list below would just be `repository_streams_list`.

Same for the other groups. wdyt @Ry-DS ?

Reply:
I tried this out but tbh not a fan because the streams files are already very large, and adding another large block at the bottom doesn't feel like it helps. Think it's fine as it is now as there is now a central location to import all streams. Thoughts?